client/analytics/settings/config.js

```diff
@@ -37,6 +37,39 @@ const orderStatuses = Object.keys( wcSettings.orderStatuses )
 	} );
 
 export const analyticsSettings = applyFilters( SETTINGS_FILTER, [
+	{
+		name: 'woocommerce_rebuild_reports_data',
+		label: __( 'Rebuild reports data:', 'wc-admin' ),
+		inputType: 'button',
+		inputText: __( 'Rebuild reports', 'wc-admin' ),
+		helpText: __(
+			'This tool will rebuild all of the information used by the reports. ' +
+				'Data will be processed in the background and may take some time depending on the size of your store.',
+			'wc-admin'
+		),
+		callback: ( resolve, reject, addNotice, refreshStoreAlerts ) => {
+			const errorMessage = __( 'There was a problem rebuilding your report data.', 'wc-admin' );
+
```

Comment:
I think we should have a TODO here to track the usage of this callback.

Reply:
Oops -- this was actually left in from a previous commit where I was attempting to refresh them on save -- but it felt a bit too hacky. This should be cleaned up.

Reply:
Removed in https://github.com/woocommerce/wc-admin/pull/1763/commits/279818019b54f63a5302eba46bb348cbd5f24ce6.